src/tutorial/impl-draft.md

```diff
@@ -54,9 +54,8 @@ Give it a try: `cargo run -- main src/main.rs` should work now!
 <aside class="exercise">
 
 **Exercise for the reader:**
-This is not the best implementation:
-It will read the whole file into memory
-– however large the file may be.
+This is not the best implementation.
+It will read the whole file into memory, no matter how large the file may be.
```

Comment:
I feel like a full stop might be too disjointed here.  Maybe a `;`?
```suggestion
This is not the best implementation;
it will read the whole file into memory, no matter how large the file may be.
```
or bridging the two parts
```suggestion
This is not the best implementation as
it will read the whole file into memory, no matter how large the file may be.
```